src/controller/src/clusters.rs

```diff
@@ -197,6 +197,16 @@ impl ReplicaLocation {
             ReplicaLocation::Unmanaged(_) => false,
         }
     }
+
+    pub fn workers(&self) -> usize {
+        let workers_per_process = match self {
+            ReplicaLocation::Managed(ManagedReplicaLocation { allocation, .. }) => {
+                allocation.workers
+            }
+            ReplicaLocation::Unmanaged(UnmanagedReplicaLocation { workers, .. }) => *workers,
+        };
+        workers_per_process * self.num_processes()
```

Comment:
(I feel like I should know this 🙈) Is this the correct way to get the number of workers for a replica?

Reply:
Looks right to me!

Reply:
ty!